salvo/src/lib/docker_volume.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+"""
+This module builds the volume mapping structure passed to a docker image 
+"""
+
+import json
+import logging
+
+# Ref: https://docker-py.readthedocs.io/en/stable/index.html
+import docker
+
+from google.protobuf.json_format import (Error, MessageToJson)
+
+from api.docker_volume_pb2 import Volume, VolumeProperties
+
+log = logging.getLogger(__name__)
+
+
+class DockerVolume():
+
+  @staticmethod
```

Comment:
Instead of marking this method static, can we define it as a module level function (i.e. remove the class DockerVolume).

We should prefer standalone functions unless having a class is really necessary. We can test for that by identifying if the class either has interesting state that changes during its lifetime or if the code needs to maintain multiple instances of the class. If we don't have good reasons for either of these two, having a standalone method will result in simpler code.

The style guide also recommends avoiding staticmethod:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#217-function-and-method-decorators

Reply:
Updated.